incidentmonitoring/EKStack/docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       - "9300"
     environment:
       BM_ES_NODE_NAME: "es-data-1"
-      ES_JAVA_OPTS: "-Xms150m -Xmx150m"
+      ES_JAVA_OPTS: "-Xms8g -Xmx8g"
```

Comment:
The doc should mention that this default Java heap requires at least 16GB (in best case 32GB) for the host machine.

Maybe this should be 1GB or 2GB by default